libraries/ESP8266SSDP/ESP8266SSDP.h

```diff
@@ -104,6 +105,7 @@ class SSDPClass{
     SSDPTimer* _timer;
     uint16_t _port;
     uint8_t _ttl;
+    uint8_t _interval;
```

Comment:
This can't be uint8_t. SSDP_INTERVAL is 1200 which is already above the max of 255 for a uint8_t.
Please change to uint32_t.